### DISCUSSION TITLE
修复funasr加载离线模型bug

### DISCUSSION THREAD

**Discussion by XiHenSuper** [modelscope/FunASR#2311] [Ideas]
```
from funasr import AutoModel
model = AutoModel(model="paraformer-zh-streaming", model_path="/root/funasr/models/iic/speech_paraformer-large_asr_nat-zh-cn-16k-common-vocab8404-online", disable_update=True)
```

按照funasr1.2.0源码python38\Lib\site-packages\funasr\download\download_model_from_hub.py第45行，加载离线模型需要定义model_path，但是定义后出现报错：`AssertionError: paraformer-zh-streaming is not registered`，排查发现**model_or_path**未正确定义，需要在funasr1.2.0源码python38\Lib\site-packages\funasr\download\download_model_from_hub.py第45行后面新增一行：`model_or_path = kwargs["model_path"]`
希望社区考虑一下更新一下吧。

**Comment by LauraGPT** [comment]
检查一下你的网络吧，或者看看文档吧。为什么不去：model="/root/funasr/models/iic/speech_paraformer-large_asr_nat-zh-cn-16k-common-vocab8404-online"
